baked_in.go

```diff
@@ -527,7 +530,28 @@ func isEthereumAddress(fl FieldLevel) bool {
 		return true
 	}
 
-	// checksum validation is blocked by https://github.com/golang/crypto/pull/28
+	// Checksum validation. Reference: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-55.md
+	digitValue := map[byte]int{
+		'0': 0, '1': 1, '2': 2, '3': 3, '4': 4,
+		'5': 5, '6': 6, '7': 7, '8': 8, '9': 9,
+		'a': 10, 'b': 11, 'c': 12,
+		'd': 13, 'e': 14, 'f': 15,
+	}
+	address = address[2:]
+	h := sha3.NewLegacyKeccak256()
+	h.Write([]byte(strings.ToLower(address)))
```

Comment:
`hash.Hash`'s `io.Writer` implementation says it never returns an error. https://golang.org/pkg/hash/#Hash

Reply:
```suggestion
        // hash.Hash's io.Writer implementation says it never returns an error. https://golang.org/pkg/hash/#Hash
	_ , _ = h.Write([]byte(strings.ToLower(address)))
```

👆 this way linters don't complain and everyone knows the error checking wasn't just missed :)